### ISSUE TITLE
Error when trying to run it on Linux.

### ISSUE THREAD

**Issue by AlFaMoDz** [kayler-renslow/arma-dialog-creator#48]
### Version and Build
1.0.7

### Summary
When I try to run the .jar this error occurs: `Error: Could not find or load main class com.kaylerrenslow.armaDialogCreator.main.ArmaDialogCreator`

### Detail
![alt text](http://i.imgur.com/BeC1tWo.png)
![alt text](http://i.imgur.com/T3i9TXl.png)
![alt text](http://i.imgur.com/SuFVPpb.png)

**Comment by kayler-renslow** [comment]
I tried this on my Linux machine and it appears it was an issue with JavaFX. Here are some steps:
1. Install Java JDK 11 (I couldn't find JavaFX for Java 8 on Linux)
> sudo apt-get install openjdk-11-jdk
2. Download this .zip and extract in your installation directory of ADC. After extraction,`start_adc.sh` should be in same directory as `adc.jar`.
3. Execute `start_adc.sh`
> sudo sh start_adc.sh

Still doesn't launch? Run `java --version` in command line. If it still says openjdk version 1.8.0_212, then you will have to modify the shell script. Change `java` at the start of the file to `/usr/lib/jvm/java-8-openjdk-amd64/bin/java` or wherever your openjdk 11 bin/java is. You could also try uninstalling java 8.

Let me know if this resolves this.

**Comment by AlFaMoDz** [comment]
Hello, first of all thank you very much for your reply.

Effectively the error was due to JavaFX and since I had several versions installed I had to change the default installation with this command `sudo update-alternatives --config java`.

Afterwards, I had to add the execution permissions to the .sh and it worked perfectly.

Thank you very much for your help and very good work!

![alt text](http://i.imgur.com/5DT63io.png)
